spec/acceptance/attributes_for_spec.rb

```diff
@@ -80,8 +80,6 @@
 describe "`attributes_for` for a class whose constructor has required params" do
   before do
     define_model("User", name: :string) do
-      def initialize(arg1, arg2)
```

Comment:
Based on the test description, I don't think we want this change. Could we add something to the body of the initialize instead to get around the rule?

Reply:
`Style/RedundantInitialize` cop is not made an offense if there is a comment in the constructor by default, so I added a comment. What do you think?
https://docs.rubocop.org/rubocop/cops_style.html#allowcomments-true-default